lib/git.js

```diff
@@ -444,10 +444,31 @@ class Git extends EventEmitter {
         callback = options;
         options = { type: 'http' };
       }
+      var enableCors = false;
+      if (typeof options.enableCors !== 'undefined') {
+        if (typeof options.enableCors === 'boolean') {
+          enableCors = options.enableCors;
+        }
+        delete options.enableCors;
```

Comment:
we should haven't have to delete this, we should check against options.cors.

Reply:
I was trying not to add any new options to the call to `createServer()`, so that's where the separate variable and deletion came from. But I'll take out all that complexity and make it simpler as you suggest. 👍